### DISCUSSION TITLE
Feature Request: Get notified when your PR develops merge conflicts.

### DISCUSSION THREAD

**Discussion by Zopolis4** [community/community#17258] [Pull Requests]
I've got a lot of outstanding pull requests, and it can get a little tedious having to check each one of them for merge conflicts, especially since there's no way to tell without clicking through on each one (#14307), so it would be nice to have a setting whereby one gets notified if their pull request develops merge conflicts.

**Comment by marc-hb** [comment]
Thanks for filing this. Linking to older and identical issues in other places. It's apparently possible if you have Slack.
- https://github.com/isaacs/github/issues/224
- https://github.com/ploomber/contributing/issues/16
